gammapy/scripts/image_fit.py

```diff
@@ -64,8 +64,11 @@ def image_fit(counts,
     log.info('Reading background: {0}'.format(background))
     sherpa.astro.ui.load_table_model('background', background)
 
-    log.info('Reading PSF: {0}'.format(psf))
-    SherpaMultiGaussPSF(psf).set()
+    if psf: 
+        log.info('Reading PSF: {0}'.format(psf))
+        SherpaMultiGaussPSF(psf).set()
+    else:
+        log.info("Not applying PSF convolution.")
```

Comment:
This is pretty uncommon for end users to not use a PSF.
Should we change this to `log.warning`?
(I'm not sure ... @adonath, you decide!)